src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.html

```diff
@@ -81,7 +81,7 @@
         </li>
         <li role="menuitem"
             *ngIf="permission.delete"
-            [ngClass]="{'disabled': !selection.hasSingleSelection || selection.first().executing}">
+            [ngClass]="{'disabled': !selection.hasSingleSelection || selection.first().cdExecuting}">
```

Comment:
Did the exact [same thing](https://github.com/ceph/ceph/pull/21614/files#diff-e863fe64385fec2409e362219933ee0aR61) before seeing this :)